README.md

```diff
@@ -1,5 +1,7 @@
 # GLWTPL (Good Luck With That Public License)
 
+[![GLWTPL](https://img.shields.io/badge/GLWT-Public_License-red.svg)](https://github.com/me-shaon/GLWTPL)
```

Comment:
Alternative: `https://badgen.net/badge/GLWT/Public%20License/red`

Maybe also include different styles, and an easy way for people to copy the badges?

Reply:
@madsmtm please make a separate PR with the alternatives.